src/libmongoc/tests/json/server_discovery_and_monitoring/integration/isMaster-command-error.json

```diff
@@ -17,7 +17,8 @@
         },
         "data": {
           "failCommands": [
-            "isMaster"
+            "isMaster",
```

Comment:
These test changes are from the corresponding [spec PR](https://github.com/mongodb/specifications/pull/969). I'll amend the commit once the spec change is approved.